MainModule/Client/UI/Default/Inspect.lua

```diff
@@ -346,10 +335,31 @@ return function(data)
 
 		addAdonisEntry("Admin Level:", data.AdminLevel, "The player's Adonis rank")
 		addAdonisEntry("Donor:", boolToStr(data.IsDonor), "Is the player an Adonis Donor?")
-		addAdonisEntry("Contributor:", boolToStr(data.IsContributor), "Is the player an Adonis Contributor (Github)?")
 		addAdonisEntry("Muted:", boolToStr(data.IsMuted), "Is the player muted? (IGNORES TRELLO MUTELIST)")
 		addAdonisEntry("Banned:", boolToStr(data.IsBanned), "Is the player banned? (IGNORES TRELLO BANLIST)")
 
+		local extrainfo = nil
+		if player:GetRankInGroup(886423) == 10 then
+			extrainfo = "User is a GitHub contributor of Adonis"
+		elseif player:GetRankInGroup(886423) == 12 then
+			extrainfo = "User is a developer of Adonis"
+		elseif player.UserId == 698712377 or player.UserId == 1237666 then
+			extrainfo = "You are inspecting the creator of Adonis"
```

Comment:
Change
`extrainfo = "You are inspecting the creator of Adonis"`
To
`extrainfo = "You are inspecting Sceleratis/Davey_Bones (Adonis creator)!`